src/components/02-molecules/11-footer/footer.njk

```diff
@@ -0,0 +1,20 @@
+<div class="columns">
+	<div class="column">
+		<a class="logo link--inverse" rel="external" href="https://platform.coop">
+			{% render '@svg', {svg: 'logo', class: 'logo__image', viewBox: '0 0 282.08 78.75'}, true %}
+			<span class="screen-reader-text">Platform Cooperativism Consortium</span>
+		</a>
+		<span id="footer-nav-label" class="screen-reader-text">footer navigation</span>
+		<ul class="nav" aria-labelledby="footer-nav-label">
+			<li><a class="link--inverse" rel="external" href="https://platform.coop/open-access-and-privacy-policy/">Our open access &amp; privacy policy</a></li>
+			<li><a class="link--inverse" rel="external" href="https://platform.coop/diversity-inclusion/">Our commitment to diversity &amp; inclusion</a></li>
+		</ul>
```

Comment:
I don't think a list is necessary here. The links are already in the Footer landmark and navigable using tab navigation.